### DISCUSSION TITLE
SSML generation – retain pronunciation context

### DISCUSSION THREAD

**Discussion by g-30** [openai/whisper#1098] [Ideas]
Would it be possible to support SSML (Speech Synthesis Markup Language) tags generation to indicate the "prosody" for every word / sentence? I.e. if the person talks fast, slow, loud, high pitch / low pitch – with SSML integration we would have that data recorded.

This would be very useful when it comes to translating one language to another with the goal of retaining the audio context so that the further TTS translation wouldn't sound like soulless wall of text read by the machine. Is it possible with Whisper?

**Comment by curiousyuvi** [comment]
Yeah that would be a great use case for whisper.

**Comment by wanghanlele12345** [comment]
+1 it would be great
